crates/order-generator/src/bin/zeth.rs

```diff
@@ -115,8 +122,16 @@ async fn main() -> Result<()> {
     let args = Args::parse();
 
     let wallet = EthereumWallet::from(args.private_key.clone());
+
+    let balance_alerts_layer = BalanceAlertLayer::new(BalanceAlertConfig {
+        watch_address: wallet.default_signer().address(),
+        warn_threshold: args.warn_balance_below,
+        error_threshold: args.error_balance_below,
+    });
```

Comment:
There are actually 2 order generators in this package, theres another one in main.rs that we should update